src/init.cpp

```diff
@@ -892,7 +895,15 @@ bool AppInit2()
     if (fServer)
         NewThread(ThreadRPCServer, NULL);
 
-    // ********************************************************* Step 12: finished
+    // ********************************************************* Step 12: start rest listenser
+
+#ifdef REST
+    uiInterface.InitMessage(_("Starting RESTful API Listener"));
+    printf("Starting RESTful API Listener\n");
+    NewThread(ThreadRESTServer, NULL);
+#endif
+
+    // ********************************************************* Step 13: finished
 
     uiInterface.InitMessage(_("Done loading"));
```

Comment:
Probably this should synchronously wait for the rest server to finish loading before saying "Done loading". We can do this with a std::promise/std::future pair. Adding a TODO comment here wouldn't be the worst idea.

Reply:
Agreed. Will make the TODO for now.